docs/organizations/using-gate-policies.md

```diff
@@ -0,0 +1,86 @@
+# Using gate policies
+
+Gate policies help you ensure that Codacy uses the same [quality gates](../repositories-configure/adjusting-quality-gates.md) across your organization repositories.
+
+Codacy provides a built-in gate policy, **Codacy Gate Policy**, which sets minimum quality levels for pull requests and commits. By default, Codacy applies this gate policy automatically to newly added repositories. You can then create new gate policies with different quality gates and make them the default for your organization.
+
+![Codacy built-in gate policy](images/gate-policy-codacy-default.png)
+
+## Creating a new gate policy {: id="creating"}
+
+To create a new gate policy for your organization:
+
+1.  Open your organization **Policies** page, tab **Gate policies**.
```

Comment:
@claudiacarpinteiro Just a reminder that we will need to update the screenshot for Coding standards and Bulk copy since they too were moving inside the Policies tab, in another PR probably, but just so we do not forget